packages/block-editor/src/components/inserter/menu.js

```diff
@@ -355,9 +357,27 @@ export class InserterMenu extends Component {
 								</a>
 							</PanelBody>
 						) }
-						{ isEmpty( suggestedItems ) && isEmpty( reusableItems ) && isEmpty( itemsPerCategory ) && (
-							<p className="editor-inserter__no-results block-editor-inserter__no-results">{ __( 'No blocks found.' ) }</p>
-						) }
+
+						<__experimentalInserterMenuExtension.Slot
+							fillProps={ {
+								onSelect,
+								onHover: this.onHover,
+								filterValue: this.state.filterValue,
+								isMenuEmpty,
```

Comment:
Can we rename this prop to better describe it does?

```suggestion
								hasItems,
```